src/components/AnnualReport/AGMDate.vue

```diff
@@ -112,193 +112,148 @@
   </v-card>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Mixins, Prop, Watch, Emit } from 'vue-property-decorator'
-import { State, Getter } from 'vuex-class'
-import { DateMixin } from '@/mixins'
+<script setup lang="ts">
+import { reactive } from 'vue'
+import { AllowableActionsComposable, DateComposable } from '@/composables'
 import { FormIF } from '@/interfaces'
-
-@Component({})
-export default class AgmDate extends Mixins(DateMixin) {
```

Comment:
script setup does not export in the composition api and instead the values are used directly